src/quacc/atoms/skzcam.py

```diff
@@ -79,57 +81,47 @@ def convert_pun_to_Atoms(
                 ]
             ],
         )
-        emb_clus = emb_clus.copy() + atom.copy()
+        embedded_cluster = embedded_cluster.copy() + atom.copy()
 
-    # Centre the embedded cluster so that atom 1 is at the [0, 0, 0] position
-    emb_clus.translate(-emb_clus[0].position)
+    # Centre the embedded cluster so that atom index 0 is at the [0, 0, 0] position
+    embedded_cluster.translate(-embedded_cluster[0].position)
 
-    # Add the charge_list to the Atoms object
-    emb_clus.set_array("oxi_states", np.array(charge_list))
-    emb_clus.set_array("atom_type", np.array(atom_type_list))
+    # Add the `oxi_states` and `atom_type` arrays to the Atoms object
+    embedded_cluster.set_array("oxi_states", np.array(charge_list))
+    embedded_cluster.set_array("atom_type", np.array([atom_type_list]))
 
-    return emb_clus
+    return embedded_cluster
 
 
-def get_atom_distances(emb_clus: Atoms, centre_position: np.ndarray) -> np.ndarray:
+def get_atom_distances(embedded_cluster: Atoms, center_position: NDArray) -> NDArray:
     """
     Returns the distance of all atoms from the centre position of the embedded cluster
 
     Parameters
     ----------
-    emb_clus : Atoms
-        The ASE Atoms object containing the atomic coordinates.
+    embedded_cluster
+        The ASE Atoms object containing the atomic coordinates of the embedded cluster.
 
     Returns
     -------
-    np.ndarray
-        An array containing the distances of each atom in the Atoms object fro the cluster centre.
+    NDArray
+        An array containing the distances of each atom in the Atoms object from the cluster centre.
     """
 
-    # Get the number of atoms in the Atoms object
-    num_atoms = len(emb_clus)
-
-    # Create an empty matrix to store the distances between each pair of atoms
-    distances = np.zeros(num_atoms)
-
-    # Calculate the distances between each pair of atoms
-    for i in range(num_atoms):
-        distances[i] = np.linalg.norm(emb_clus[i].position - centre_position)
-
-    return distances
+    return np.array([np.linalg.norm(atom.position - center_position) for atom in embedded_cluster])
```

Comment:
This was an area that also was ripe for an easy refactor. Note you also weren't returning an array before but rather a list.